commit fae9f462e31c935c973850bae1d38d5fe69a082c (refs/changes/12/4612/2)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-07-31T17:52:11-04:00 (1 year, 2 months ago)
    
    TRITON-653 vm-agent TRITON-628 bug - missing jsprim dependency
    Reviewed by: Rui <rui@joyent.com>
    Approved by: Rui <rui@joyent.com>

diff --git a/package.json b/package.json
index c60ac42..cc9fcad 100644
--- a/package.json
+++ b/package.json
@@ -10,6 +10,7 @@
         "cueball": "2.1.1",
         "deep-diff": "0.3.3",
         "eslint": "2.13.1",
+        "jsprim": "2.0.0",
         "kthxbai": "0.4.0",
         "lstream": "0.0.4",
         "mockery": "1.4.0",
